stage1/modules/classes-prototypes/README.md

```diff
@@ -35,30 +36,29 @@
   - Extends intro and usage
   - Mixins intro and usage
 
-## Приблизительное время прохождения модуля
+## Approximate Module Completion Time ⏱️
+
+6 hours
 
-6 часов
+## Theory 📖
 
-## Теория
+Students are encouraged to study the following resources:
 
-Посмотреть вебинары:
+- [FreeCodeCamp video about JavaScript Classes](https://youtube.com/watch?v=2ZphE5HcQPQ)
+- [RS School Presentaton "JavaScript Classes"](https://kirilknysh.github.io/js-classes-talk/#/)
+- [JavaScript Classes](https://javascript.info/classes)
+- [Prototypes and Inheritance](https://javascript.info/prototypes)
 
-- [RS School. JavaScript Classes](https://youtu.be/O462BetYst8)
-- [Презентация из вебинара "RS School. JavaScript Classes"](https://kirilknysh.github.io/js-classes-talk/#/)
-- [JavaScript Classes & Prototypes / Классы и прототипы в JavaScript. JS/FE 2021Q3](https://youtu.be/QceUxKQeFDA)
-- [Classes / Классы](https://youtu.be/uy_6CeZ6MTU)
+## Practice 💻
 
-## Практика
+- [Codewars tasks](https://github.com/rolling-scopes-school/tasks/blob/master/tasks/codewars/Codewars-2022Q3-OOP.md)
+- Complete the test "Classes & Prototypes" in the RS APP > Auto Test
 
-- [Задачи на Codewars](https://github.com/rolling-scopes-school/tasks/blob/master/tasks/codewars/Codewars-2022Q3-OOP.md)
-- Вам необходимо пройти тест "Classes & Prototypes" в RS APP > Auto Test
+## Additional Resources 📘
 
-## Дополнительные материалы
+Expand your knowledge with these additional materials:
 
 - [Composition over Inheritance](https://www.youtube.com/watch?v=wfMtDGfHWpA)
-- http://mrale.ph/v8/resources.html
-- [JS Core by Dmitry Soshnikov, 2nd edition](http://dmitrysoshnikov.com/ecmascript/javascript-the-core-2nd-edition-rus/)
-- [JS Core by Dmitry Soshnikov](http://dmitrysoshnikov.com/ecmascript/ru-javascript-the-core/)
-- [Объекты, Learn.JavaScript.ru](https://learn.javascript.ru/object-basics)
-- [Прототипы, наследование, Learn.JavaScript.ru](https://learn.javascript.ru/prototypes)
-- [Классы, Learn.JavaScript.ru](https://learn.javascript.ru/classes)
+- [MDN article "Using classes"](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_classes)
+- [Inheritance and the prototype chain](http://developer.mozilla.org/en-US/docs/Web/JavaScipt/Inheritance_and_the_prototype_chain)
```

Comment:
Broken link 
- [Inheritance and the prototype chain](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Inheritance_and_the_prototype_chain)

Reply:
Fixed